cpp/examples/parameter_sampling.cpp

```diff
@@ -18,8 +18,8 @@
 * limitations under the License.
 */
 #include "memilio/utils/parameter_distributions.h"
-#include "secir/parameter_space.h"
-#include "secir/secir.h"
+#include "ode_secir/parameter_space.h"
+#include "ode_secir/model.h"
 
```

Comment:
maybe please also rename file to ode_secir_*

Reply:
done